_includes/payment-order-other-features.md

```diff
@@ -1305,6 +1307,7 @@ elements. When in doubt, please follow the [robustness principle](https://en.wik
 [http-api-problems]: https://tools.ietf.org/html/rfc7807
 [image_disabled_payment_menu]: /assets/img/checkout/test-purchase.png
 [image_enabled_payment_menu]: /assets/img/checkout/guest-payment-menu-450x850.png
+[merchant-identified-payer]: /payment-menu/other-features#merchant-identified-payer
```

Comment:
Won't linking directly to `payment-menu` here be a problem in the Checkout section where I assume `payment-order-other-features.md` is also used?